paperparcel/src/main/AndroidManifest.xml

```diff
@@ -1 +1 @@
-<manifest package="paperparcel"/>
\ No newline at end of file
+<manifest package="paperparcel.core"/>
```

Comment:
Why the package name change here?

Reply:
Updated android gradle plugin says "paperparcel" is incorrect package name.

Reply:
Because this is a library project this should not change anything I suppose?

Reply:
I think we should undo this change unless one of us clearly understands what the tools were complaining about

Reply:
I really can't because it's impossible to compile library after that. Here is error:
```
> Task :paperparcel:verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':paperparcel:verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /Users/bejibx/Documents/GitHub/paperparcel/paperparcel/build/intermediates/aapt_friendly_merged_manifests/release/aapt/AndroidManifest.xml:2: error: attribute 'package' in <manifest> tag is not a valid Android package name: 'paperparcel'.

```

I think the reason is package name attribute is closely tied to application id attribute and application id has convention that it must contain at least two segments (see android documentation [here](https://developer.android.com/studio/build/application-id)). Well, package name only affects where ```R``` class is generated and how to reference application components in manifest (Activities, Services, etc...).

Well, If you still concerned about that, I can undo android plugin update. I also have to undo gradle wrapper update to version 4, because old android plugin using some method removed from gradle version 5.

Reply:
Yep, you can see this check [here](https://android.googlesource.com/platform/frameworks/base.git/+/master/tools/aapt2/link/ManifestFixer.cpp#154), and implementation is [here](https://android.googlesource.com/platform/frameworks/base.git/+/master/tools/aapt2/util/Util.cpp#169).

It Looks like this:
```C
bool IsAndroidPackageName(const StringPiece& str) {
    return IsAndroidNameImpl(str) > 1 \|\| str == "android";
}

static int IsAndroidNameImpl(const StringPiece& str) {
  int pieces = 0;
  for (const StringPiece& piece : Tokenize(str, '.')) {
    if (piece.empty()) {
      return -1;
    }
    const char first_character = piece.data()[0];
    if (!::isalpha(first_character)) {
      return -1;
    }
    bool valid = std::all_of(piece.begin() + 1, piece.end(), [](const char c) -> bool {
      return ::isalnum(c) || c == '_';
    });
    if (!valid) {
      return -1;
    }
    pieces++;
  }
  return pieces;
}
```